src/client/components/region-banner-component/banner.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
```

Comment:
There is no need to have such long naming. Instead of `components/region-banner-component/banner.jsx`, please, name it as `components/region-banner/index.jsx` and follow this convention from now on.